dhis-2/dhis-api/src/main/java/org/hisp/dhis/relationship/RelationshipService.java

```diff
@@ -74,6 +74,15 @@
      */
     Relationship getRelationship( long id );
 
+    /**
+     * Checks for the existence of a relationship by UID. Takes into account
```

Comment:
Maybe

`Checks if relationship for given UID exists (including deleted relationships).`